OpenRA.Mods.Common/Traits/World/Locomotor.cs

```diff
@@ -292,7 +293,8 @@ bool CanMoveFreelyInto(Actor actor, Actor ignoreActor, CPos cell, CellConditions
 			if (ignoreActor == null && blocking.Overlaps(actor.Owner.PlayerMask))
 				return false;
 
-			if (cellBlocking.HasCellBlocking(CellBlocking.Crushable))
+            // We can crush this actor and it is not moving
+			if (cellBlocking.HasCellBlocking(CellBlocking.Crushable) && !cellBlocking.HasCellBlocking(CellBlocking.IsMoving))
```

Comment:
Why would it matter if the blocking actor is moving when we can crush it anyway?